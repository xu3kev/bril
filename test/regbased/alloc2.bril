../regalloc/alloc2.out