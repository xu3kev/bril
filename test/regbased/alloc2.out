2
9
8
#load: 9
#store: 13
